Makefile

```diff
@@ -61,4 +61,7 @@ run-docker:
 exec_app:
 	cd lib/app; bundle _$(shell cat .bundler-version)_ exec ruby console_app.rb
 
+exec_command:
+	cd lib/app; bundle _$(shell cat .bundler-version)_ exec ruby console_app.rb $1
```

Comment:
Trying to understand `exec_command`. Is this the CLI we talked about? So the expectation of the CLI is to provide an `index_name`, select a connector, and start syncing. It bypasses the regular syncing mechanism, and we won't know that a sync has been run from `.elastic-connectors`.

Reply:
It's not used (yet). I was thinking of passing the commands from the command line, then I did it by gets, so this one is kinda extra. We might decide on another approach. I just needed something to run the sync with, some CTA, and this was what came to mind first but might or might not be used later. I guess that for scripting, it does make sense to be able to run the commands with params from command line; for user interactions aka manual mode, `gets` could still come in handy. But those are just different options.